downgrade

```diff
@@ -448,12 +425,41 @@ parse_options() {
 }
 
 cli() {
-  parse_options "$@"
+  declare -a system_args
+
+  # get system and command-line arguments and parse everything
+  IFS=' ' read -r -a system_args <<<"$(read_downgrade_conf)"
+  parse_options "${system_args[@]}" "$@"
+
+  # make arrays unique
+  IFS=' ' read -r -a terms <<<"$(printf "%s\n" "${terms[@]}" | uniq | tr '\n' ' ')"
+  IFS=' ' read -r -a PACMAN_CACHE <<<"$(printf "%s\n" "${PACMAN_CACHE[@]}" | uniq | tr '\n' ' ')"
+
+  # proceed with rest of workflow
   main "${terms[@]}"
   pacman -U "${pacman_args[@]}" "${to_install[@]}"
   prompt_to_ignore "${to_ignore[@]}"
 }
 
+# Set script defaults
```

Comment:
You dropped the "Up-convert older variables", which is fine, but I think would mean a major version bump?

Reply:
Yes, since we are going away from user-inputted environmental variables altogether; I thought it would be good to remove these as well.

Major version bump sounds appropriate.